### PR TITLE
Revisión Maintenance curation.es.rmd es auto

### PR DESCRIPTION
**Pull request by paocorrales**
I did the R2 without knowing there was a new commit from @maelle fixing the newline problem. I had merge issues and decided to keep my version with the human revision. I'm sorry we lost the changes from the new commit. 

Otherways is ready @yabellini

### CODE REVIEW

File: maintenance_curation.es.Rmd [hunk 1]
```diff
@@ -1,116 +1,143 @@
...
+No se comprobarán en el sistema de rOpenSci.
 
-- Los paquetes archivados no se mostrarán por defecto en la página web de paquetes. Una pestaña especial de las páginas de paquetes mostrará éstos con `"type": "archived"` que fueron revisados por pares o mantenidos por el personal.
+- Los paquetes archivados no se mostrarán por defecto en la página web de paquetes.
+Una pestaña especial de páginas de paquetes los mostrará con `"type": "archived"`, hayan sido revisados por pares o mantenidos por el personal.
 
-- Los paquetes archivados pueden ser desarchivados cuando el antiguo o un nuevo mantenedor esté dispuesto a solucionar los problemas y quiera revivir el paquete. Para ello, por favor [ponte en contacto con rOpenSci](https://ropensci.org/contact/). Se transfieren a la organización ropenscilabs.
+- Los paquetes archivados pueden desarchivarse cuando alguien nuevo, o quien lo mantenía anteriormente, esté dispuesto a solucionar los problemas y quiera revivirlo.
```
Comment (yabellini) [inline maintenance_curation.es.Rmd:49]:
```suggestion
- Los paquetes archivados pueden desarchivarse cuando alguien nuevo/a, o quien lo mantenía anteriormente, esté dispuesto/a a solucionar los problemas y quiera revivirlo. 
```

File: maintenance_curation.es.Rmd [hunk 2]
```diff
@@ -1,116 +1,143 @@
...
+- Si quien mantienen un paquete no responde consultas por un mes, o si ésta persona lo pide, rOpenSci puede buscar una nueva persona para mantener determinados paquetes revisados por pares que considere que tienen un alto valor para la comunidad en función de la base de personas que lo usan (descargas), dependencias inversas o los objetivos estratégicos de rOpenSci. 
 
-- Cuando se archiven, estos paquetes pasarán de la organización "ropensci" de GitHub a la organización "ropensci-archive" (o cuentas GitHub de autor si se desea), siguiendo [guía de transferencia](#archivalguidance). Ganarán el tipo "archivado" en el registro. Conservarán las etiquetas y etiquetas de "revisado por pares". No se construirán en el sistema rOpenSci.
+- Cuando se archiven, estos paquetes pasarán de la organización "ropensci" de GitHub a la organización "ropensci-archive" (o a las cuentas de GitHub quienes los mantienen si lo desean), siguiendo [guía de transferencia](#archivalguidance).
```
Comment (yabellini) [inline maintenance_curation.es.Rmd:78]:
```suggestion
- Cuando se archiven, estos paquetes pasarán de la organización "ropensci" de GitHub a la organización "ropensci-archive" (o a las cuentas de GitHub quienes los mantienen si así lo desean), siguiendo la [guía de transferencia](#archivalguidance). 
```

File: maintenance_curation.es.Rmd [hunk 3]
```diff
@@ -1,116 +1,143 @@
...
+- Semestral o anualmente, rOpenSci se pondrá en contacto con quienes mantienen los paquetes incubados con repositorios de al menos tres meses de antigüedad para consultar por el estado de desarrollo y las preferencias para la migración a revisión por pares, ropensci-archive, o a sus organizaciones.
+En base a la respuesta, el paquete se migrará inmediatamente, se iniciará la revisión por pares, o la migración se aplazará hasta la próxima revisión. 
+Los paquetes incubados se migrarán a ropensci-archive por defecto después de un año, siguiendo la [guía de transferencia](#archivalguidance).
 
-- Los paquetes de incubadora archivados ganarán el tipo "archivado".
+- Los paquetes incubados archivados ganarán la etiqueta "archived".
 
-### Paquetes no R de la incubadora
+### Paquetes incubados no de R
```
Comment (yabellini) [inline maintenance_curation.es.Rmd:119]:
```suggestion
### Paquetes incubados que no son de R
```

**Review by yabellini — APPROVED** [review]
Minor changes

**Comment by yabellini** [comment]
@maelle let me know if I can merge this or if you will try to apply the changes you did and then merge.

**Comment by maelle** [comment]
My fix made the automatic translation less gibberish but now that the translation review is done, it's fine to merge I think :slightly_smiling_face:

**Pull request merged**